hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMNodeInfo.java

```diff
@@ -55,12 +55,12 @@
 public class SCMNodeInfo {
```

Comment:
Please add `@Immutable`, and `import net.jcip.annotations.Immutable`.  (HDDS-10741)

Reply:
Added.